MessageBoxEnhance/wpfApplication1/Window2.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<Window x:Class="WpfApplication1.Window2"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:WpfApplication1"
+        mc:Ignorable="d"
+        Title="Previous_Window" Height="300" Width="416.807">
+    <Grid HorizontalAlignment="Left" Height="253" Margin="10,10,0,0" VerticalAlignment="Top" Width="389">
+        <TextBlock x:Name="textBlock" HorizontalAlignment="Left" Margin="10,6,0,0" TextWrapping="Wrap" Text="   Please Select Your Programming Language." VerticalAlignment="Top" Height="45" Width="379"/>
+        <ComboBox x:Name="comboBox" HorizontalAlignment="Left" Margin="256,6,0,0" VerticalAlignment="Top" Width="123" RenderTransformOrigin="0.58,0.037">
+            <ComboBoxItem Content="C" HorizontalAlignment="Left" Width="203"/>
+            <ComboBoxItem Content="C++" HorizontalAlignment="Left" Width="203"/>
```

Comment:
ComboBoxItem also should be added in the cs file. It should be like dynamic. We should not hard code the content here.